src/Symfony/Component/Form/Extension/Core/DataMapper/RadioListMapper.php

```diff
@@ -26,23 +26,17 @@
 class RadioListMapper implements DataMapperInterface
 {
     /**
-     * @var ChoiceListInterface
-     */
-    private $choiceList;
-
-    public function __construct(ChoiceListInterface $choiceList)
-    {
-        $this->choiceList = $choiceList;
-    }
-
-    /**
      * {@inheritdoc}
      */
-    public function mapDataToForms($data, $radios)
+    public function mapDataToForms($choice, $radios)
     {
+        if (!is_string($choice)) {
```

Comment:
@webmozart I re-check the code and here the view data should always be a string.

In contrary, since `mapsFormsToData()` happens before transformation we should accept `null` there as well.

Do you agree or am I missing something ? Thanks.